cmd/nodecmd/validatePrimaryNetwork.go

```diff
@@ -30,38 +28,18 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var (
-	deployTestnet   bool
-	deployMainnet   bool
-	keyName         string
-	subnetName      string
-	useLedger       bool
-	ledgerAddresses []string
-	weight          uint64
-	duration        time.Duration
-
-	ErrMutuallyExlusiveKeyLedger = errors.New("--key and --ledger,--ledger-addrs are mutually exclusive")
-	ErrStoredKeyOnMainnet        = errors.New("--key is not available for mainnet operations")
-	ErrNoBlockchainID            = errors.New("failed to find the blockchain ID for this subnet, has it been deployed/created on this network?")
-)
-
-func newJoinCmd() *cobra.Command {
+func newValidatePrimaryNetworkCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "join [clusterName]",
-		Short: "(ALPHA Warning) Join a subnet as a validator",
+		Use:   "validatePrimaryNetwork [clusterName]",
+		Short: "(ALPHA Warning) Join Primary Network as a validator",
 		Long: `(ALPHA Warning) This command is currently in experimental mode.
 
-The node join command enables all nodes in a cluster to be validators of a Subnet.
-The command also enables nodes to be Primary Network validators by omitting the --subnet flag.
-If The command is run before the nodes are Primary Network validators, the command will first
-make the nodes Primary Network validators before making them Subnet validators. 
-If The command is run before the nodes are bootstrapped on the Primary Network, the command will fail. 
-You can check the bootstrap status by calling avalanche node status <clusteName>`,
+The node validatePrimaryNetwork command enables all nodes in a cluster to be validators of Primary
+Network.`,
```

Comment:
Just curious, what happens if you call this command multiple times? What if you call it, add another node, then call it again?